tests/tracking/test_model_registry.py

```diff
@@ -13,6 +13,7 @@
 
 from mlflow.entities.model_registry import RegisteredModel
 from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE, ErrorCode
```

Comment:
looks like this is failing in lint as unused imports

Reply:
Yeah, I'm running through the tests again right now. I'll ping you once it's fixed.